### PR TITLE
Update zap jar

### PR DESCRIPTION
**Pull request by psiinon**
Note that this is a pre release version for testing - I'll update the PR
with the final jar once all changes have been merged.

### CODE REVIEW

**Comment by psiinon** [comment]
Notr surprised this has failed - will push another update

**Comment by kingthorin** [comment]
Want me to tackle those deprecations?

**Comment by psiinon** [comment]
yeah, if you have time :)
I'm going to raise PRs for the other branches too, but right now my eclipse is tied up generating new language files ;)

**Comment by kingthorin** [comment]
Got it

**Comment by psiinon** [comment]
Ah, the changed add-ons should also have their versions incremented :/ I'm doing that for the alpha PR now...

**Comment by kingthorin** [comment]
Ok, will do

**Comment by kingthorin** [comment]
`Test org.zaproxy.zap.extension.ascanrules.TestParameterTamperUnitTest FAILED`
Investigating

File: src/org/zaproxy/zap/extension/pscanrules/ZapAddOn.xml [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 	<author>ZAP Dev Team</author>
 	<url/>
 	<changes>
 	<![CDATA[
-	Fix false positive with Secure Pages Include Mixed Content and JavaScript files (Issue 3581).<br>
-	Fix false positive with Private IP Disclosure when target is a private IP on a non-standard port (Issue 3549).<br>
-	Fix false positive with X-Content-Type-Options Header Missing with certain system locales.<br>
-	Fix X-Content-Type-Options help content (Issue 3986).<br>
-	Remove N/A value from parameter of alert Session ID in URL Rewrite.<br>
+	Minor code changes to address deprecation.<br>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrules/ZapAddOn.xml]: `Update for 2.7.0`? We also need to update the minimum ZAP version.

**Comment by psiinon** [comment]
Note that we've decided not to update the core, so there PRs can be approved and merged when they are ready

**Comment by kingthorin** [comment]
Squash it when I merge it?

**Comment by psiinon** [comment]
Please :)

**Pull request merged**